script.js

```diff
@@ -33,8 +33,35 @@ const specials = {
 		USD: 3,
 	},
 }
-  
 
+let currencyRate = [];
+
+function getData() {
+  fetchCurrencyRate()
+    .then((responce) => {
+ 
+      currencyRate.push(...responce)
+      displayTable(responce);
+      displayConverter(responce);
+
+    })
+    .catch((error) => console.error(error))
+}
+
+getData();
```

Comment:
In software development, it's really important to be as descriptive as possible. Since the `getData` function is doing more than just retrieving data (it's also displaying a table and converter), I would recommend renaming it to something more descriptive, like `getAndShowData`, to clarify the function's purpose.